### PR TITLE
is it correct to replace link to quickstart with template?

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ The components within this repository are delivered as multi-targeted Nuget pack
...
+- `Jet.ConfluentKafka.FSharp` [![Jet.ConfluentKafka.FSharp NuGet](https://img.shields.io/nuget/vpre/Jet.ConfluentKafka.FSharp.svg)](https://www.nuget.org/packages/Jet.ConfluentKafka.FSharp/): Wraps `Confluent.Kafka` to provide efficient batched Kafka Producer and Consumer configurations, with basic logging instrumentation. Used in the [`propulsion project kafka`](dotnet-tool-provisioning--benchmarking-tool) tool command; see [`dotnet new proProjector -k; dotnet new proConsumer` to generate a sample app](https://github.com/jet/dotnet-templates#propulsion-related) using it (see the `BatchedAsync` and `BatchedSync` modules in `Examples.fs`).
```
Comment (bartelink) [inline README.md]: > [`propulsion project kafka`](dotnet-tool-provisioning--benchmarking-tool)

should probably link to https://github.com/jet/propulsion#dotnet-tool-provisioning--projections-test-tool (there is no doc in there that describes the tool's sub-commands as such)

**Comment by bartelink** [comment]
Thanks for the PR, and apologies for the error in the first place - yes, your suggestion is bang on.

I made a suggestion on the code; there is a further broken link on the same line.

File: README.md (hunk 1 above)
Comment (dzmitry-lahoda) [reply README.md]: i am reading documentation, will update what will found.

Comment (bartelink) [reply README.md]: Thanks - I for one really appreciate those sorts of things being picked up
And a heads up, I've just released nugets for all and sundry in the last few hours if you're playing about

**Comment by bartelink** [comment]
Ping; would like to fix the link and remove the hanging PR ;)

**Comment by dzmitry-lahoda** [comment]
i will close for now and send broader pull request when get there, i temporally work on other part of codes than event sourcing and stopped to read thought all docs for now.

**Comment by bartelink** [comment]
👍 applied your changes; thanks for catching

**Pull request closed without merging**